caravel/views.py

```diff
@@ -1653,10 +1695,11 @@ class CssTemplateModelView(CaravelModelView, DeleteMixin):
     category_label=__("Sources"),
     category_icon='')
 
-appbuilder.add_link(
-    'SQL Lab <span class="label label-danger">alpha</span>',
-    href='/caravel/sqllab',
-    icon="fa-flask")
+appbuilder.add_link('SQL Editor',
+href='/caravel/sqllab', icon="fa-flask", category='SQL Lab')
+appbuilder.add_link('Query Search',
+
+href='/caravel/sqllab?search', icon="fa-flask", category='SQL Lab')
```

Comment:
you have to bind an even to hash change:
https://developer.mozilla.org/en-US/docs/Web/API/WindowEventHandlers/onhashchange